.github/workflows/apidoc.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        apiGroup: [ 'public-api', 'management-api', 'control-api' ]
+        apiGroup: [ 'default-api', 'public-api', 'management-api', 'control-api' ]
```

Comment:
I don't like the name "default-api", IMO "observability-api" would be better. The bigger problem is, that in practice it will not (yet) work, because SwaggerHub has a limit of 3 APIs on the free tier which we are using. 

We have contacted SmartBear, but so far we have yet to receive a response.

Reply:
Ok, then let's leave this PR open/stale until there is an answer from them.